lib/sms-sender-ids.json

```diff
@@ -0,0 +1,5 @@
+[
+  [ "CA", "16474909977" ],
+  [ "GB", "Firefox" ],
+  [ "US", "15036789977" ]
```

Comment:
Hrm, not sure about putting this in the `config.js` anymore since it could get pretty big. Despite that, seems a little strange to be hanging out in the lib directory. Maybe pull into config dir?